content/en/user-guide/aws/events/index.md

```diff
@@ -120,6 +120,21 @@ $ localstack logs
 ...
 {{< /command >}}
 
+## Supported target types
+
+At this time LocalStack supports the following target types for EventBridge rules:
```

Comment:
```suggestion
At this time LocalStack supports the following [target types](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-targets.html#eb-console-targets) for EventBridge rules:
```
Add the link to the AWS list: https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-targets.html#eb-console-targets

Reply:
I hope this is the correct AWS link :) and thank you for adding it in.